### PR TITLE
Replace blspy with chia_rs

### PR DESCRIPTION
**Pull request by AmineKhaldi**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
     long_description = fh.read()
 
 dependencies = [
-    "blspy>=0.9",
+    "chia_rs==0.3.3",
```
Comment (altendky) [inline setup.py:9]: Can we be a bit looser?  Otherwise this has to update in lock step with blockchain.

**Review by altendky — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (AmineKhaldi) [reply setup.py]: Done.

**Comment by emlowe** [comment]
I think we could do with >=0.2.13 - that is the chia_rs version used by chia 2.1.2 where we switched to using it for BLS

**Comment by AmineKhaldi** [comment]
> I think we could do with >=0.2.13 - that is the chia_rs version used by chia 2.1.2 where we switched to using it for BLS

Done.

**Review by richardkiss — APPROVED** [review]

**Pull request merged**